### ISSUE TITLE
Why I run it with cifar10 ,but the result is not fine

### ISSUE THREAD

**Issue by fengchegithub** [pytorch/examples#452]
I run it with the same network structure and set the epoch=40,but why the result is not well enough.Another question is that why you set the random seed？

**Comment by msaroufim** [comment]
> but why the result is not well enough

Could just be try more epochs and if it's still bad then try changing other hyperparameters or your model

> Another question is that why you set the random seed？

The random seed is fixed so that the results you get are always the same which is helpful to guarantee convergence, reproducibility and sane behavior in tests
